README.md

```diff
@@ -9,19 +9,38 @@ This repo contains packages and a CLI for analyzing the network connectivity of
 $ ./bin/vpcanalyzer -h
 Usage of vpc-network-config-analyzer:
   -analysis-type string
-        supported analysis types: all_endpoints,all_subnets,single_subnet, diff_all_endpoints, diff_all_subnets (default "all_endpoints")
+        supported analysis types:
+        0) all_endpoints  - supported with: txt, json, md, drawio, arch_drawio, debug,
+        1) all_subnets  - supported with: txt, json,
+        2) single_subnet  - supported with: txt,
+        3) diff_all_endpoints  - supported with: txt, md,
+        4) diff_all_subnets  - supported with: txt, md,
+         (default "all_endpoints")
+  -debug
+        run in debug mode
   -format string
-        output format; must be one of txt,md,drawio,arch_drawio,json (default "txt")
+        output format; must be one of:
```

Comment:
let's keep the section of output format as was originally... so that the cli options are not over detailed

Reply:
![image](https://github.com/np-guard/vpc-network-config-analyzer/assets/152963690/0ff76309-bbe0-4e5e-b00c-878c455a5c43)
it seems good and readable what do you think

Reply:
yes, though I do think the extra details in the `format` section are redundant in a way, and we should prefer to have more succinct description